### DISCUSSION TITLE
Localizing Permission Names

### DISCUSSION THREAD

**Discussion by quasiperfect** [chiiya/filament-access-control#30] [Q&A] [answered]
Hi

I tried to add my permissions translations to `lang\vendor\filament-access-control\en.json` but I don't see any change in the admin panel
what I'm doing wrong?

**Comment by chiiya** [comment, marked as answer]
Should work when using the default translation files, e.g. `lang/en.json`

**Reply by quasiperfect** [reply]
if i edit the vendor\chiiya\filament-access-control\resources\lang\en.json it works if i edit the published files it doesn't

**Reply by quasiperfect** [reply]
for some reason, i had the files in the wrong location lang\vendor\filament-access-control\ instead of resources\lang\vendor\filament-access-control

i moved the files and now all is ok
